Updates on Week02

AlexVOiceover opened the pull request:
Improved worflows on GitHub Projects Workshop
Updated links for Thought of the week
Added Mentors to schedule
Added Resources (Github Branches interactive tuto, basic npm youtube tuto).
Added the chatbot Project Description to the site (it was in Notion)

ShaughnAnderson94 reviewed (changes requested):
you probably know a lot of this stuff but heres some bits that look weird 
says not updated and one item randomly bold
![image](https://github.com/user-attachments/assets/0bb0e463-1dd6-4ccd-825b-d22d178e91f6)
weird gap around talk
![image](https://github.com/user-attachments/assets/7621d1f1-6057-453a-a495-b85f4db3dda5)
workshop has no link to the workshop also are you doing a checkin in the morning?
![image](https://github.com/user-attachments/assets/d125398b-84e6-46ec-a21c-3332613f71b2)

otherwise looks good

AlexVOiceover commented:
Fixed Topics, I removed everything that was already in the schedule.
Fixed gap before talks, I wasn't sure about starting time
Workshops with no links. They have the links on the overview. I would like them to feel like they can spend the time as they want, as long as they do all the workshops. So the idea is more like "Time allocated for workshops" instead of specifics.
Yeah, I would like to do checkins on Fridays, is that too harsh?

ShaughnAnderson94 reviewed (approved):
looking lots nicer
